### PR TITLE
docs(qrcode): download demo add background color

### PR DESCRIPTION
**Pull request by OriginRing**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [x] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

---

### <samp>🤖 Generated by Copilot at cb4d865</samp>

Improved QR code demo by using `theme` and `useToken` from `antd` to set background color.

### <samp>🤖 Generated by Copilot at cb4d865</samp>

* Import `theme` module and `useToken` hook from `antd` to access theme tokens ([link](https://github.com/ant-design/ant-design/pull/43645/files?diff=unified&w=0#diff-ab3dd275135e8b301922b3a09c2699b03129eadd7be8f840e565cec5124361faL1-R5))
* Set `bgColor` prop of `QRCode` component to `token.colorWhite` to match theme and improve contrast ([link](https://github.com/ant-design/ant-design/pull/43645/files?diff=unified&w=0#diff-ab3dd275135e8b301922b3a09c2699b03129eadd7be8f840e565cec5124361faL17-R31))

### CODE REVIEW

**Comment by afc163** [comment]
The reason to update this demo?

**Comment by OriginRing** [comment]
> 

目前下载后是透明的，我本想看下是否可以支持带边距下载，发现 `qrcode.react` 将在下个版本支持 `marginSize`，就补充了背景色

**Comment by afc163** [comment]
用例挂了，请检查。

File: components/qr-code/demo/download.tsx [hunk 1]
```diff
@@ -14,13 +16,17 @@ const downloadQRCode = () => {
...
-      Download
-    </Button>
-  </div>
-);
+const App: React.FC = () => {
+  const { token } = useToken();
+
+  return (
+    <div id="myqrcode">
+      <QRCode value="https://ant.design/" bgColor={token.colorWhite} style={{ marginBottom: 16 }} />
```
Comment (afc163) [inline components/qr-code/demo/download.tsx:24]:
```suggestion
      <QRCode value="https://ant.design/" bgColor="#fff" style={{ marginBottom: 16 }} />
```

是不是这样就好了？

Comment (OriginRing) [reply components/qr-code/demo/download.tsx]: 这样也可以

**Review by afc163 — APPROVED** [review]

**Pull request merged**